tests/integration/auth/edge-cases.test.ts

```
@@ -0,0 +1,503 @@
+/**
```

ESLint

[error] Definition for rule '@typescript-eslint/ban-types' was not found.

(@typescript-eslint/ban-types)